phoenicis-javafx/src/test/java/org/phoenicis/javafx/AppConfigurationInitialisationTest.java

```diff
@@ -1,26 +1,37 @@
-/**
- *
- */
 package org.phoenicis.javafx;
 
 import org.junit.Test;
 import org.springframework.context.annotation.AnnotationConfigApplicationContext;
 import org.testfx.api.FxToolkit;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
 import java.util.concurrent.TimeoutException;
 
+import static org.junit.Assert.fail;
+
 /**
  * A test for to check that the JavaFX application can be correctly started
  *
  * @author marc
  */
 public class AppConfigurationInitialisationTest {
 
+    /**
+     * run the test in a temporary application root environment
+     */
+    private static void setupApplicationRoot() throws IOException {
+        Path tempDir = Files.createTempDirectory(AppConfigurationInitialisationTest.class.getName());
+        tempDir.toFile().deleteOnExit();
+        System.setProperty("application.user.root", tempDir.toString());
```

Comment:
This sounds like something we want to do in a `@BeforeClass` annotated method.

Reply:
Why should it be different than the setup for headless?

Reply:
It shouldn't.
I think we should move the headless initialization to a method annotated with `@BeforeClass` too.
I don't know why I didn't do that initially...

Reply:
Maybe because I wrote that method ;) (you had it in maven).